components/dashboard/src/settings/Account.tsx

```diff
@@ -7,23 +7,40 @@
 import { User } from "@gitpod/gitpod-protocol";
 import { useContext, useState } from "react";
 import Modal from "../components/Modal";
+import { getGitpodService, gitpodHostUrl } from "../service/service";
 import { UserContext } from "../user-context";
 import { SettingsPage } from "./SettingsPage";
 
 export default function Account() {
     const { user } = useContext(UserContext);
 
     const [modal, setModal] = useState(false);
+    const [typedEmail, setTypedEmail] = useState('');
+
+    const primaryEmail = User.getPrimaryEmail(user!);
+
+    const deleteAccount = async () => {
+        await getGitpodService().server.deleteAccount();
+        //TODO we are sad to see you go view
+        document.location.href = gitpodHostUrl.asApiLogout().toString();
+    };
 
     const close = () => setModal(false);
     return <div>
         <Modal visible={modal} onClose={close}>
             <h3 className="pb-2">Delete Account</h3>
             <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-2">
-              <p className="mt-1 mb-2 text-base">Are you sure you want to delete your account? This action will remove all the data associated with your account in Gitpod and cannot be reversed.</p>
+                <p className="py-4 text-gray-900 text-base">You are about to permanently delete your account.</p>
+                <ol className="text-gray-400 text-sm list-outside list-decimal">
+                    <li className="ml-5">All your workspaces and related data will be deleted and cannot be restored afterwards.</li>
+                    <li className="ml-5">Your subscription will be cancelled. If you obtained a Gitpod subscription through the GitHub marketplace, you need to cancel your plan there.</li>
+                </ol>
+                <p className="pt-4 pb-2 text-gray-600 text-base font-bold">Type your email to confirm</p>
+              <input className="border-2 border-gray-400 w-full" type="text" placeholder={primaryEmail} onChange={e => setTypedEmail(e.target.value)}></input>
             </div>
             <div className="flex justify-end mt-6">
-                <button className="border-red-900 bg-red-500 hover:bg-red-700" onClick={close}>Delete Account</button>
+                <button className="text-gray-900 border-gray-100 bg-gray-100 hover:bg-gray-200 hover:border-gray-400" onClick={close}>Cancel</button>
```

Comment:
**suggestion**: Let's remove the background here for the default state and add the `font-medium` class which is missing also in some other buttons, too.

```suggestion
                <button className="text-gray-500 border-white bg-white hover:bg-gray-100 hover:border-gray-100 font-medium" onClick={close}>Cancel</button>
```

Reply:
I'm adding `text-base` to the default button style

Reply:
Oh, wait. Buttons use `14px` text. The should be using `font-medium`, too.

Reply:
Actually, button already had text-sm applied. Is that not right?

Reply:
Yes, `text-sm` is corrent. Font weight (`font-medium`) is missing. I tried this in https://github.com/gitpod-io/gitpod/pull/3521#discussion_r598168129 but didn't work. Maybe I've missed something. ❓